### PR TITLE
Cash 14

### PR DESCRIPTION
**Pull request by BoulderBrains**

### CODE REVIEW

**Comment by BoulderBrains** [comment]
FYI @ksekhri

**Comment by BoulderBrains** [comment]
@mcstover & @ksekhri 
There were a couple misses with this ticket. 
- Currently graphql doesn't handle the removal of a team loan attribution. This ticket was created to close the loop on that issue: https://kiva.atlassian.net/browse/CASH-264 (once this ticket is done this no team attribution should work)
- Right now the only thing not styled correctly is the background-image within the team attribution dropdown. Right now it's #484848 aka $charcoal and that's hardcoded to the image. I'm connecting with Adam shortly about the best way forward for that issue. Everything else is done.

**Comment by BoulderBrains** [comment]
Just followed up with Adam and requested a new image for the dropdown of the right color. Going to keep this branch live until I get that asset and add it in. Just waiting for new image now.

File: src/components/Checkout/TeamAttribution.vue [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+.team-select {
+	float: left;
+	max-width: rem-calc(250);
+}
+
+.team-select-dd {
+	border: 1px solid $gray;
+	border-radius: $button-radius;
+	height: rem-calc(40);
+	background-image: url('../../assets/images/customDropdown.png');
```
Comment (mcstover) [inline src/components/Checkout/TeamAttribution.vue]: please format like so...
`background-image: url('~@/assets/images/backgrounds/tipbar-bg-small.jpg');`

**Pull request merged**